scanpy/external/pl.py

```diff
@@ -234,3 +236,90 @@ def sam(
         if colorbar:
             plt.colorbar(cax, ax=axes)
     return axes
+
+
+@_doc_params(show_save_ax=doc_show_save_ax)
+def wishbone_marker_trajectory(
+    adata: AnnData,
+    markers: List[str],
+    no_bins: int = 150,
+    smoothing_factor: int = 1,
+    min_delta: float = 0.1,
+    show_variance: bool = False,
+    figsize: Optional[Tuple[float, float]] = None,
+    return_fig: bool = False,
+    show: bool = True,
+    save: Optional[Union[str, bool]] = None,
+):
+    """\
+    Plot marker trends along trajectory, and return trajectory branches for further
+    analysis and visualization (heatmap, etc..)
+
+    Parameters
+    ----------
+    adata
+        Annotated data matrix.
+    markers
+        Iterable of markers/genes to be plotted.
+    show_variance
+        Logical indicating if the trends should be accompanied with variance.
+    no_bins
+        Number of bins for calculating marker density.
+    smoothing_factor
+        Parameter controlling the degree of smoothing.
+    min_delta
+        Minimum difference in marker expression after normalization to show
+        separate trends for the two branches.
+    figsize
+        width, height
+    return_fig:
+        Return the matplotlib figure.
+    {show_save_ax}
+
+    Returns
+    -------
+    Updates `adata` with the following fields:
+
+    `Trunk_wishbone` : :class:`pandas.DataFrame` (`adata.uns`)
+        Computed values before branching
+    `Branch1_wishbone` : :class:`pandas.DataFrame` (`adata.uns`)
+        Computed values for the first branch
+    `Branch2_wishbone` : :class:`pandas.DataFrame` (`adata.uns`)
+        Computed values for the second branch.
```

Comment:
Farther down, you use lowercase-only names, and I think that’s the convention in the rest of scanpy too.

For easier autocompletion, those three should be lowercase as well.

Reply:
fixed and changed to lowercase.